src/views/partials/nav.njk

```diff
@@ -1,31 +1,14 @@
-{% set sections = [
-    {
-        pages: [
-            {
-                url: "/",
-                title: "Getting started"
-            }
-        ]
-    }, {
-        title: "Styles",
-        pages: collections.styles | eleventyNavigation
-    }, {
-        title: "Components",
-        pages: collections.components | eleventyNavigation
-    }
-] %}
-
-<nav>
+<nav class="hidden md:block md:py-6">
     <ul>
         {% for section in sections %}
             <li>
                 {% if section.title %}
                     <h2 class="font-normal text-base text-grey-2 mb-2">{{section.title}}</h2>
                 {% endif %}
-                <ul class="mb-4">
+                <ul class="md:mb-4">
                     {% for post in section.pages %}
                         {% if page.url == post.url %}
-                            <li class="mb-2 pl-2 -ml-3 font-bold border-l-4 border-ocean-blue" aria-current="page">
+                            <li class="mb-2 pl-2 md:-ml-3 font-bold border-l-4 border-ocean-blue" aria-current="page">
```

Comment:
Should this <li> be closed there are three opening ones now and only two closing

Reply:
I know it looks weird but the logic is:

```
{% if page.url == post.url %}
    <li class="mb-2 pl-2 md:-ml-3 font-bold border-l-4 border-ocean-blue" aria-current="page">
{% else %}
    <li class="mb-2">
{% endif %}
        <a href="{{post.url | url }}" class="no-underline hover:underline">{{post.title}}</a>
    </li>
{% endfor %}
```

So there's only ever one opening `<li>` due to the if/else. It's not the cleanest way of doing it anyway, I'll see if I can spread the attributes or something instead.

Reply:
Changed how this works so it's much cleaner syntax now